src/pages/Fox/components/RFOXSliders.tsx

```diff
@@ -26,24 +31,63 @@ const DEFAULT_SHAPESHIFT_REVENUE = 10000
 const MAX_SHAPESHIFT_REVENUE = 1000000
 const MIN_SHAPESHIFT_REVENUE = 0
 
+const inputStyle = {
+  input: {
+    fontWeight: 'bold',
+    textAlign: 'right',
+  },
+}
+
 export const RFOXSliders: React.FC<RFOXSlidersProps> = ({
   shapeShiftRevenue,
   setShapeShiftRevenue,
   depositAmount,
   setDepositAmount,
   maxDepositAmount,
 }) => {
+  const handleShapeShiftRevenueChange = useCallback(
+    (values: NumberFormatValues) => {
+      setShapeShiftRevenue(bnOrZero(values.value).toNumber())
+    },
+    [setShapeShiftRevenue],
+  )
+
+  const debounceShapeShiftRevenuChange = useMemo(
+    () => debounce(handleShapeShiftRevenueChange, 1000),
+    [handleShapeShiftRevenueChange],
+  )
+
+  const handleDepositAmountChange = useCallback(
+    (values: NumberFormatValues) => {
+      setDepositAmount(bnOrZero(values.value).toNumber())
+    },
+    [setDepositAmount],
+  )
+
+  const debounceDepositAmountChange = useMemo(
+    () => debounce(handleDepositAmountChange, 1000),
+    [handleDepositAmountChange],
+  )
+
   return (
     <VStack height='100%' spacing={8} mt={6}>
       <Stack spacing={4} width='full'>
-        <Flex width='full' justifyContent='space-between' fontWeight='medium'>
+        <Flex width='full' justifyContent='space-between' alignItems='center' fontWeight='medium'>
           <Text translation='foxPage.rfox.depositAmount' />
-          <Amount.Crypto
-            value={depositAmount.toString()}
-            symbol='FOX'
-            fontWeight='bold'
-            maximumFractionDigits={0}
-          />
+
+          <Box width='35%' sx={inputStyle}>
+            <NumberFormat
+              decimalScale={2}
+              customInput={Input}
+              isNumericString={true}
+              suffix={' FOX'}
+              decimalSeparator={'.'}
+              inputMode='decimal'
+              thousandSeparator={','}
+              value={depositAmount.toString()}
+              onValueChange={debounceDepositAmountChange}
```

Comment:
q: Why do we debounce this? Slider changes are reflected automatically in inputs, but input changes are not, which looks like a bug for users